src/Console/Command/DocumentationCommand.php

```diff
@@ -68,6 +69,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return 0;
     }
 
+    /**
+     * @param FixerInterface[] $fixers
```

Comment:
`list<FixerInterface>` would be better. `FixerInterface[]` is an alias for `array<int|string, FixerInterface>`

Reply:
Would list suggest that the type of index is not defined or not of interest, does it also implies the order of the elements is trivial? Would be nice to have some documentation backing this up as I think it is not in PSR5?

Reply:
`list<T>` means `array<int, T>` where keys start at 0 and are consecutive without gaps.